docs/arch/adr-31.md

```diff
@@ -0,0 +1,90 @@
+# ADR 31: RotatingRectangle Tool and TemporalRotatingRectangle Tool
+
+## Status
+
+In Review
+
+## Context
+
+The Solar Jets project requires a TemporalRotatingRectangle drawing tool for volunteers to capture the width, height, angle and time properties of a solar jet. At the time of this writing, the `front-end-monorepo` has a standard Rectangle drawing tool which gives the width and height properties. Based on the standard Rectangle drawing tool, we want to create a RotatingRectangle drawing tool and a TemporalRotatingRectangle drawing tool.
+
+The RotatingRectangle drawing tool is a common tool used in the Panoptes code based and eventually needs to be written in the `front-end-monorepo`. This is a good opportunity to complete this task.
```

Comment:
```suggestion
The RotatingRectangle drawing tool is a common tool used in the Panoptes code base and eventually needs to be written in the `front-end-monorepo`. This is a good opportunity to complete this task.
```

Reply:
Change `Panoptes` to `Panoptes-Front-End` for clarity here too.